### DISCUSSION TITLE
where is the script to put in the console

### DISCUSSION THREAD

**Discussion by fzazdbl** [hxr404/Discord-Console-hacks#77] [Q&A] [answered]
This may sound a bit silly on my part but I don't know where to find this scipt I need to put in the console is there a kind person to help me?

**Comment by hxr404** [comment, marked as answer]
Click expand under Console Hacks
and then expand under the script you want to paste

For example:
![image](https://user-images.githubusercontent.com/55095883/219885012-b7eef8be-71ef-4bc7-96d8-4d0ff83b3ce6.png)


<hr>

_Disclaimer: If you are not sure about what you are doing I don't recommend proceeding, as using these scripts can possibly get your account banned (or worse).
If you paste scripts without knowing what they do, you basically trust the author of the script 100% to not do anything malicious (token grabbers can be hidden in a single line of code).
Even though I can assure you that no script form this repo contains token loggers, unless you know a bit of js or know someone who does, you have to take my word for this. And there are many strangers out there trying to scam you so be careful._
Other than that: have fun fucking around with discord :D

**Reply by fzazdbl** [reply]
thank you very much you are very kind
